requirements/base.txt

```diff
@@ -10,7 +10,7 @@ opencv-python>=4.5.3.56
 pandas>=1.1.0
 pytorch-lightning>=1.6.0,<1.7.0
 timm==0.5.4
-torchmetrics>=0.9.0,<=0.9.1
+torchmetrics==0.9.1
```

Comment:
The latest torchmetrics version is 0.9.3. Would it be an idea to do something like 

```
torchmetrics>=0.9.1,<=0.9.3
```